SimpleRss/SimpleRss/ViewControllers/FeedViewController.swift

```diff
@@ -90,7 +89,9 @@ extension FeedViewController: UITableViewDelegate {
         
         guard let cell = tableView.cellForRow(at: indexPath) as? FeedViewCell else { return }
         
-        cell.descriptionLabel.numberOfLines = cell.descriptionLabel.numberOfLines == 0 ? 1 : 0;
+        let feed = feedList[indexPath.row]
+        feed.toggle()
+        cell.expanding(isExpanded: feed.isExpanded)
```

Comment:
I believe that cell will be reloaded after calling `tableView.beginUpdates(); tableView. endUpdates()`, so `cell.expanding(isExpanded: feed.isExpanded)` is not really needed here

Reply:
cell.expanding is existed for change style for label, which allows make description text wider in 
the cell